content/html/concepts/elements/terms/hr/hr.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: '<hr>'
+Description: 'Used to draw represent a semantic break between text elements.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<hr>`** element is used to represent a semantic break between text elements. For example, a switch in topic.
+
+**Note:** The **`<hr>`** tag is only for semantic distinction. Best practice to create a horizontal line for presentational purposes is to use [CSS](https://www.codecademy.com/resources/docs/css).
```

Comment:
```suggestion
> **Note:** The **`<hr>`** tag is only for semantic distinction. Best practice to create a horizontal line for presentational purposes is to use [CSS](https://www.codecademy.com/resources/docs/css).
```

Our current preference is to set off **Note** items in a blockquote.

Reply:
It might be worth it to explain past usage vs. current usage. We have many historical notes like that through the HTML elements entries.